### ISSUE TITLE
X-link

### ISSUE THREAD

**Issue by Carreau** [jdfreder/notebook_cherry_picker#1]
To IPython main issue :-)

https://github.com/ipython/ipython/issues/3777

BTW, the link to your youtube video is broken :-)

**Comment by jdfreder** [comment]
Ahh thanks @Carreau !

Glad to see something like this will be officially supported. I've updated the youtube video links, go ahead and check them out :)

**Comment by Carreau** [comment]
> Glad to see something like this will be officially supported.

Yeah, yo uhad your earplug when we spoke of it. don't you remember voting for `tags` vs `label` vs `labels` vs `id` vs `name` ?

> I've updated the youtube video links, go ahead and check them out :)

I had it to work on my computer :-)

**Comment by jdfreder** [comment]
>  remember voting for tags vs label vs labels vs id vs name

Ahhh, that's right.  I totally forgot about that

**Comment by jdfreder** [comment]
I'll try to keep track of [your IPEP](https://github.com/ipython/ipython/wiki/IPEP-20%3A-Unformal-structure-Of-Metadata-1)
